GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/LooseObjectStepTests.cs

```diff
@@ -73,6 +113,13 @@ private void DeleteFiles(List<string> filePaths)
             }
         }
 
+        private int CountPackFiles()
+        {
+            return Directory.GetFiles(this.PackRoot)

```

Comment:
Why did you not use a search pattern in this call like `*.pack`?  Is that not case insensitive?

Reply:
*.pack should work i'll update